danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/models/gitlab/GitLab.kt

```diff
@@ -108,9 +108,9 @@ data class GitLabMergeRequest(
 @Serializable
 data class GitLabMergeRequestTimeStats(
     @SerialName("human_time_estimate")
-    val humanTimeEstimate: Int?,
+    val humanTimeEstimate: Int? = 0,
     @SerialName("human_time_spent")
-    val humanTimeSpent: Int?,
+    val humanTimeSpent: Int? = 0,
```

Comment:
Does this needs to be Int? given we have a default that is not nil? (same the one before)